sapl/relatorios/forms.py

```diff
@@ -68,6 +69,68 @@ def __init__(self, *args, **kwargs):
         )
 
 
+class RelatorioVotacoesNominaisFilterSet(django_filters.FilterSet):
+
+    @property
+    def qs(self):
+        parent = super(RelatorioVotacoesNominaisFilterSet, self).qs
+        return parent.distinct().order_by('-votacao_id', 'parlamentar')
+
+    class Meta(FilterOverridesMetaMixin):
+        model = VotoParlamentar
+        fields = ['data_hora']
+
+    def __init__(self, *args, **kwargs):
+        super(
+            RelatorioVotacoesNominaisFilterSet, self
+        ).__init__(*args, **kwargs)
+
+        self.filters['data_hora'].label = 'Período (Data Inicial - Data Final)'
+
+        tipo_materia = '''<div class="col-md-6"><div id="div_id_tipo_materia" class="form-group"><label for="id_tipo_materia" class="col-form-label ">
+               Tipo de Matéria Legislativa
+           </label><div class=""><select name="tipo_materia" class="select form-control" id="id_tipo_materia"><option value="" selected="">---------</option>'''
+        for tipo in TipoMateriaLegislativa.objects.all():
+            tipo_materia += '<option value="' + str(tipo.id) + '">' + tipo.descricao + '</option>'
```

Comment:
FYI, a legibilidade desse comando pode ser melhorada com o uso de f-strings, conforme abaixo. 

O conselho é que se dê preferência ao uso de f-strings sempre que possível ao invés da concatenação de strings. **Embora, conforme o comentário neste review, `tipo_materia` deva ser transformado em field da classe.**

```suggestion
            tipo_materia += f'<option value="{tipo.id}">{tipo.descricao}</option>'
```